### PR TITLE
Document running Trino in a container

### PR DESCRIPTION
**Pull request by nineinchnick**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?
improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
docs

> How would you describe this change to a non-technical end user or system administrator?
n/a

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by mosabua** [comment]
I am happy to have the client tools in a separate client folder .. make sense imho. In the past we decided not to do that because it breaks incoming links. Personally I think that is not a problem and the navigation makes the files easy to find anyway.

**Comment by nineinchnick** [comment]
@mosabua I can try setting up redirects for moved pages using this https://documatt.gitlab.io/sphinx-reredirects/usage.html

**Comment by mosabua** [comment]
> @mosabua I can try setting up redirects for moved pages using this https://documatt.gitlab.io/sphinx-reredirects/usage.html

Personally I don't think that is necessary.

File: docs/src/main/sphinx/installation/containers.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Once you are done with your exploration, enter the ``quit`` command.
+
+Alternatively, you can use the Trino CLI installed directly on your workstation.
+The default server URL in the CLI of http://localhost:8080 matches the port used
+in the command to start the container. More information about using the CLI can
+be found in :doc:`/client/cli`. You can also connect with any other client
+application, such as `DBeaver <https://dbeaver.io/>`_, using
+the :doc:`/client/jdbc`.
+
+If you try to execute a query and see an error message like
```
Comment (mosabua) [inline docs/src/main/sphinx/installation/containers.rst:66]: I would get rid of this paragraph and more until "configuring Trino"

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor adjustments and then good to go.

File: docs/src/main/sphinx/installation/containers.rst (hunk 1 above)
Comment (nineinchnick) [reply docs/src/main/sphinx/installation/containers.rst]: I removed it, except for one sentence that the status initially shows `(health: starting)`, which I moved above when the `docker ps` command is first introduced.

**Comment by nineinchnick** [comment]
@mosabua all done; CC @hashhar @findepi

**Review by mosabua — APPROVED** [review]
Looks good!

**Review by raunaqmorarka — APPROVED** [review]

**Pull request merged**